### DISCUSSION TITLE
Consiglio per videocamere

### DISCUSSION THREAD

**Discussion by Capirex666** [indomus/forum#1083] [HUB: Home Assistant]
Buongiorno.
Mi potreste consigliare delle telecamere Ezviz (dato che ne ho altre) da esterno, a batteria e con protocollo RTSP per integrarle a Home Assistant? Ho comprato Le BC1 con base ma da quanto mi ha detto l'utente KC-inDomus non hanno il protocollo RSTP e quindi dovrò fare il reso su amazon. 
Vi ringrazio e buona giornata a tutti
Alessio Giovannini

**Comment by NdR91** [comment]
Probabilmente il problema dipende proprio dal fatto che sono a Batteria. Infatti, della gamma Ezviz, credo che BC1 sia l'unico modello senza flusso rtsp esposto.

**Reply by Capirex666** [reply]
Quindi secondo te cosa dovrei comprare?

**Reply by NdR91** [reply]
Sono nella tua stessa situazione. Volevo sostituire le mie Yi Cam con delle EZVIZ, ma quelle che ho preso utilizzano il codec h265, non compatibile.

A questo punto ho preso delle altre Yi Cam (ho preso il modello U PRO), che poi flasherò con fw yi hack
